sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/enums/SystemCapabilityType.java

```diff
@@ -260,6 +260,28 @@
 	 */
     AUDIO_PASSTHROUGH,
 
+	/**
+	 * Available Synchronously after Register App Interface response <br>
+	 * <b>Note:</b> @SuppressWarnings("unchecked") may be needed when casting depending on implementation
+	 * <table border="1" rules="all">
+	 * 		<tr>
+	 * 			<th>Enum Name</th>
+	 * 			<th>Return Type</th>
+	 * 			<th>Description</th>
+	 * 			<th>Requires Async?</th>
+	 * 			<th>Notes</th>
+	 * 		</tr>
+	 * 		<tr>
+	 * 			<td>PCM_STREAMING</td>
+	 * 			<td>AudioPassThruCapabilities</td>
```

Comment:
Can you add this to the larger list above in the comments as well?

Reply:
done